platform/platform_android.go

```diff
@@ -3,20 +3,10 @@
 
 package platform
 
-type Platform struct{}
-
-const name = "platform"
-
-func (self *Platform) Name() string {
-	return name
-}
-
 func (self *Platform) Collect() (result interface{}, err error) {
-	result, err = getPlatformInfo()
-	return
+	return nil, nil
```

Comment:
Doesn't this change mean now that we don't return any data when using the old API since the original code would fill out the `result` var returned? If so, this seems like a breaking change.

Reply:
`getPlatformInfo` is just bellow and return `nil` all the time.